ostelco-ios-client/Extensions/UIWindow+ShakeToOpenHelp.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  UIWindow+ShakeToOpenHelp.swift
+//  ostelco-ios-client
+//
+//  Created by mac on 6/8/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+extension UIWindow {
+    open override func motionEnded(_ motion: UIEvent.EventSubtype, with event: UIEvent?) {
```

Comment:
@havard024 @sgoodwin I would really, really strongly recommend against overriding a function in an extension - you can wind up with undefined or unpredictable behavior. With the app delegate method, did it work if you used the RootCoordinator's top view controller instead of just the root view controller?